### ISSUE TITLE
make a strong concluding claim rather than vague "may" claim

### ISSUE THREAD

**Issue by castedo** [tskit-dev/what-is-an-arg-paper#442]
Feel free to close this issue.

This concluding sentence strikes me as needing clarification to be more useful to readers:

https://github.com/tskit-dev/what-is-an-arg-paper/blob/55986e675fdd2db3bf1687e0ac75e107c67b318b/paper.tex#L1834-L1836

I'm usually a fan of caveats and conveying uncertainty but the "may" here seems an indication that the concluding claim is not focused enough.

My feedback here comes from a baseline belief that tskit WILL ease software burden for researcher with data well encoded by gARG. Not a _may_ ease, but _will_ ease with a reasonable assumption of contemporary research software competency.

The ambiguity of the meaning of community standard adoption seems to me to weaken the strength of the claim from "will" to "may". I can agree there might be uncertainty as to how many researchers have data that is well encoded by gARG. But assuming a researcher has data that is well encoded by gARG, then it seems totally reasonable to hard advocate for tskit and claim is will, not may, easy software burden.

**Comment by castedo** [comment]
"well-encoded by gARG" is probably a bad description. Perhaps something more along the lines of "research data that benefits from gARG encoding". Perhaps this leads to a bit of a self-referential claim. But my hunch is you can clarify the a scope of research data analysis needs (benefiting from gARG) and then hard claim the tskit software is awesome within that scope.

**Comment by jeromekelleher** [comment]
It's a tricky bit of social engineering - I completely agree with you, but finishing up on strong claims may annoy some :shrug:

**Comment by jeromekelleher** [comment]
I think we'll keep the current slightly wishy-washy text, in the hope it avoids irritating the people we're trying to influence.

**Comment by hyanwong** [comment]
I agree with @jeromekelleher . A slightly alternative wording could go something like: "Adopting it as a community standard has the potential to ease software implementation....", but I have no strong feelings about that vs what we already have.
